database/pebbledb/db.go

```diff
@@ -64,7 +67,7 @@ type Config struct {
 }
 
 // TODO: Add metrics
-func New(file string, configBytes []byte, log logging.Logger, _ prometheus.Registerer) (database.Database, error) {
+func New(file string, useSyncWrites bool, configBytes []byte, log logging.Logger, _ prometheus.Registerer) (database.Database, error) {
```

Comment:
I think the `useSyncWrites` should be part of the config.

Reply:
updated